woocommerce/woocommerce
# WP-CLI support for WooCommerce

This PR WIP.
## Testing
- Make sure you've [wp-cli](http://wp-cli.org/) installed.
- Checkout this branch.
- If everything set up properly, subcommand `wp wc` should be working.
  
  ```
  $ wp wc
  usage: wp wc coupon <command>
  or: wp wc customer <command>
  or: wp wc order <command>
  or: wp wc product <command>
  or: wp wc report <command>
  
  See 'wp help wc <command>' for more information on a specific command.
  ```
## Features
- [x] Coupon
  - [x] Create
  - [x] Types -- lists all coupon types
- [x] Customer
  - [x] Create
  - [x] Downloads -- lists all downloads of a customer
  - [x] Orders -- lists all orders of a customer
- [x] Order
  - [x] Create
  - [x] Category -- composite commands
  - [x] Create
  - [x] Types -- lists all coupon types
- [x] Report
  - [x] List -- lists available reports
  - [x] Sales
  - [x] Top Sellers
  - [x] Clear transients

/cc @mikejolley @mattyza
